### ISSUE TITLE
Automatically stopped after generating the code

### ISSUE THREAD

**Issue by Ershu1** [Helixform/CodeCursor#44]
Why is it that when the code is generated, the generation is automatically stopped when the code has not been generated. And in the lower right corner, a button showing whether to accept the generation is complete

**Comment by macasas** [comment]
Yep, that's all I get too.  Your code has been generated, Accept or Reject, and nothing is every put into the editor.  Keys are added, model selected.  So basically cannot get this extension to do anything.
